api/src/backend/api/v1/views.py

```diff
@@ -1008,6 +1027,24 @@ def inserted_at_to_uuidv7(self, inserted_at):
             return None
         return datetime_to_uuid7(inserted_at)
 
+    def get_serializer_class(self):
+        if self.action == "findings_services_regions":
+            return FindingDynamicFilterSerializer
+
+        return super().get_serializer_class()
+
+    def get_filterset_class(self):
+        if self.action in ["findings_services_regions"]:
+            self.ordering_fields = []
```

Comment:
This is covered in `get_ordering_fields` right? Or is it necessary here as well?

Reply:
I remove get_ordering_fields, this functions doesn't work for the doc generation